### ISSUE TITLE
Versioning : color contrast in dark mode with red and green background

### ISSUE THREAD

**Issue by woluweb** [joomla/joomla-cms#47838]
### Steps to reproduce the issue

Articles > Versions button > Compare

### Version

6.1

### Expected result

No matter whether you are in Dark Mode or Light Mode, the Compare popup of Versioning should have a good color contrast

### Actual result

With dark mode, the changes are

- white on light green
- white on light red
and is not readable (color contrast)

See screenshots

<img width="992" height="412" alt="Image" src="https://github.com/user-attachments/assets/2293d1b4-d5d5-46bc-9ac8-4f827129148c" />
<img width="992" height="412" alt="Image" src="https://github.com/user-attachments/assets/66e1a190-61ac-4bf8-9b7c-976a99e1c088" />

### What device were you using? (eg, phone, laptop)

desktop

### What web browser were you using? (eg, Chrome, Safari)

Firefox (latest)

### Additional Comments

1.
Maybe `contrast-color()` could make sense to solve this kind of issues.
https://developer.mozilla.org/en-US/docs/Web/CSS/Reference/Values/color_value/contrast-color

2.
I noticed this issue on Versions > Compare and I don't think right now of other places using that red/green background, but maybe there are other places to fix in the backend

3.
When editing from the front-end, the issue does not arise because front-end edition does not follow the light/dark mode setting of the backend. I also checked the case of a front-end Template which is in dark mode, but even in that case the Modal Window for Versions Compare has white background
